concent_api/core/admin.py

```diff
@@ -24,21 +25,25 @@ def queryset(self, request, queryset):
         elif self.value() == 'passive':
             passive_state_names = [x.name for x in Subtask.PASSIVE_STATES]
             return queryset.filter(state__in=passive_state_names)
+        elif self.value() == 'active_downloads':
+            current_timestamp = get_current_utc_timestamp()
+            return Subtask.objects_with_timing_columns.filter(time_for_download_file__gte=current_timestamp)
```

Comment:
The issue said to add a filter that shows both active tasks and tasks with active downloads at the same time. Your filter shows only one or the other.